data/saifrahmn.json

```diff
@@ -0,0 +1,43 @@
+{
+    "name": "Saif Rahman",
+    "type": "personal",
+    "bio": "Engineer-Student-Enthusiast",
+    "avatar": "https://github.com/saifrahmn.png",
```

Comment:
Avatar is not required, please read the docs for the fields https://linkfree.eddiehub.io/docs/quickstart